src/jsMain/kotlin/io/beatmaps/user/account.kt

```diff
@@ -163,6 +173,65 @@ class AccountComponent : RComponent<AccountComponentProps, AccountComponentState
                     }
                 }
             }
+            div("mb-3") {
+                label("form-label") {
+                    attrs.htmlFor = "description"
+                    +"Description"
+                }
+                textarea(classes = "form-control") {
+                    key = "description"
+                    attrs.id = "description"
+                    attrs.value = state.description
+                    attrs.rows = "5"
+                    attrs.onChangeFunction = {
+                        setState {
+                            description = descriptionRef.current?.value ?: ""
+                        }
+                    }
+                    ref = descriptionRef
+                }
+                div("d-grid") {
+                    button(classes = "btn btn-success", type = ButtonType.submit) {
+                        attrs.onClickFunction = { ev ->
+                            ev.preventDefault()
+
+                            if (props.userDetail.description == state.description) {
+                                setState {
```

Comment:
I'd probably just ignore this tbh, the user wants this as their description and it is.